lib/rubies/game.rb

```diff
@@ -198,6 +187,25 @@ def game
       end
       byebye
     end
-  end
 
+    private
+
+    def check_answer(current, input, target)
```

Comment:
Unused method argument - `current`. If it's necessary, use `_` or `_current` as an argument name to indicate that it won't be used.